Makefile

```
@@ -289,6 +292,7 @@ bundle: manifests operator-sdk kustomize ## Generate bundle manifests and metada
 	sed -i.bak "s/createdAt:.*/`grep -oP 'createdAt:.*' config/manifests/bases/$(BUNDLE_PACKAGE).clusterserviceversion.yaml`/" ./bundle/manifests/*.yaml && rm ./bundle/manifests/*.bak
 	sed -i.bak 's|containerImage: .*|containerImage: '${IMG}'|' ./bundle/manifests/*.yaml && rm ./bundle/manifests/*.bak
 	sed 's/annotations://' config/metadata/$(BUNDLE_PACKAGE).annotations.yaml >> bundle/metadata/annotations.yaml
+	sed -i "s~^ version: \".*\"~ version: \"${VERSION}\"~" config/metadata/activemq-artemis-operator.annotations.yaml
```

The file `config/metadata/$(BUNDLE_PACKAGE).annotations.yaml` is used in the line `sed 's/annotations://' config/metadata/$(BUNDLE_PACKAGE).annotations.yaml >> bundle/metadata/annotations.yaml`
```suggestion
	sed -i "s~^ version: \".*\"~ version: \"${VERSION}\"~" config/metadata/$(BUNDLE_PACKAGE).annotations.yaml
	sed 's/annotations://' config/metadata/$(BUNDLE_PACKAGE).annotations.yaml >> bundle/metadata/annotations.yaml
```